starter_code/index.html

```diff
@@ -7,12 +7,17 @@
 		<title>Weather Background</title>
 		<link href='http://fonts.googleapis.com/css?family=Open+Sans:300italic,300,600,800' rel='stylesheet' type='text/css'>
 
+
+
 		<!-- external CSS link -->
 		<link rel="stylesheet" href="css/reset.css">
 		<link rel="stylesheet" href="css/style.css">
 		<script src="http://ajax.googleapis.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
+
 	</head>
 	<body>
+		<!--external JS link -->
+		<script src="js/main.js"></script>
```

Comment:
@Defy1x looks like js file is not correctly linked up because none of the events are working from my end. can you try adding the script inside head tag? Thanks!